gp_mobo/acquisition_funcs/expected_improvement.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+from scipy import stats
+
+from gp_mobo.acquisition_funcs.hypervolume import Hypervolume
+
+
+def expected_improvement(pred_means: np.ndarray, pred_vars: np.ndarray, y_best: float) -> np.ndarray:
+    """
+    Calculate the expected improvement of a set of predictions.
+    Args:
+        pred_means: The predicted means of the model.
+        pred_vars: The predicted variances of the model.
+        y_best: The best value observed so far.
+
+    Returns:
+        np.ndarray: The expected improvement of the predictions.
+    """
+    std = np.sqrt(pred_vars)
```

Comment:
If std is zero the division would be by zero, not sure if we need to handle this, seems pretty unlikely?

Reply:
Should still handle it and pop some tests in